ADAL/src/telemetry/ADTelemetryHttpEvent.m

```diff
@@ -85,18 +85,7 @@ - (void)addAggregatedPropertiesToDictionary:(NSMutableDictionary*)eventToBeDispa
     }
     [eventToBeDispatched setObject:[NSString stringWithFormat:@"%d", httpEventCount] forKey:AD_TELEMETRY_HTTP_EVENT_COUNT];
     
-    NSArray* properties = [self getProperties];
-    for (ADTelemetryProperty* property in properties)
-    {
-        if ([property.name isEqualToString:AD_TELEMETRY_HTTP_RESPONSE_CODE]
-            ||[property.name isEqualToString:AD_TELEMETRY_OAUTH_ERROR_CODE]
-            ||[property.name isEqualToString:AD_TELEMETRY_HTTP_ERROR_DOMAIN]
-            ||[property.name isEqualToString:AD_TELEMETRY_HTTP_PATH]
-            ||[property.name isEqualToString:AD_TELEMETRY_HTTP_REQUEST_ID_HEADER])
-        {
-            [eventToBeDispatched setObject:property.value forKey:property.name];
-        }
-    }
+    //TODO: output http event properties to aggregated event
```

Comment:
How to output http properties to aggregated event remains undecided according to the design doc.